source/administration/changelog.md

```diff
@@ -4,6 +4,146 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+
+**IMPORTANT:** If you upgrade from a release earlier than 5.27, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).
+
+### Highlights
+
+#### New admin roles to delegate administration tasks to other types of administrators (E20)
+ - New admin roles are additional system roles that have access to designated areas of the System Console. This enables you to delegate certain administrative tasks to other members of your organization.
+
+#### Certificate-based authentication with AD/LDAP (E10)
+ - You can now improve the security of your AD/LDAP authentication with certificate-based  AD/LDAP authentication.
+
+#### Stay current with in-product notices
+ - Now with in-product notices, users and Admins will be made aware of the newest product enhancements from within Mattermost. [Learn more about in-product notices here](https://about.mattermost.com/default-notices/).
+
+### Improvements
+
+#### User Interface (UI)
+ - Improved the readability of the toast banner message timestamp, post timestamp, and date separators.
+ - Added animation for emoji reactions on webapp.
+ - Added the ability to use ``Ctrl + B`` and ``Ctrl + I`` to add bold and italics markdown formatting to selected text.
+ - Clicking on original message creator's username in discontinuing posts now opens the user's profile popover.
+ - Added support for PSD file preview.
+ - When the ``Enable Latex Rendering`` option is set to ``true``, the current code now doesn't highlight.
+ - Updated the UX of the ``More unreads`` indicator in the channel sidebar.
+ - ``Select Team`` list container now scales in width based on browser window width.
+ - Added support for signaling login to other tabs (Windows, macOS and Linux browsers).
+
+#### Search
+ - Added wildcard support to Bleve.
+ - Search terms including stopwords now return matching stopwords instead of an empty result.
+ - Removed duplication in ``is_or_search`` and ``IncludeDeletedChannels`` parameters for search.
+ - ``*`` characters are now filtered from the search terms in the database.
+ - Fixed inconsistencies across product when using ``in:@` / `in:``, such as displaying Direct and Group Messages in ``in:@`` search suggestions.
+
+#### Notifications
+ - Added an option in the **Account Settings** to select different desktop notification sounds.
+
+#### Command Line Interface (CLI)
+ - Added ``config migrate``, ``config subpath``, ``user delete``, ``integrity``, ``user migrate_auth``, ``moveChannel``, ``updateChannelPrivacy``, ``restoreTeam``, ``channel delete``, and plugin marketplace commands to mmctl.
+ 
+#### Plugins
+ - Plugins now start concurrently on server startup.
+ - Plugin tooltips are now only rendered when user hovers over a link.
+ - Added a ``CreateCommand`` plugin API that creates a slash command that is not handled by the plugin itself.
```

Comment:
@cpoile Should we also include ListCommands, etc. in here?  https://github.com/mattermost/mattermost-server/blob/31ed3b8a3eb003fc77c8bbcabf60d244c297beb5/plugin/api.go#L984